Documentation/contributing/testing/ci.rst

```diff
@@ -221,6 +224,16 @@ For a full list of Jenkins Jobs, see `Jenkins
 <https://jenkins.cilium.io/view/PR/>`_. Trigger phrases are configured within
 each job's build triggers advanced options.
 
+``test-older-k8s`` allows testing of all K8s versions 1.14 through 1.19 (it will trigger the appropriate ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs).
+If a specific K8s version fails, it can be re-ran using ``test-x.xx-4.9`` (with ``x.xx`` being the specific K8s version).
+
+``test-missed-k8s`` is deprecated and superseded by ``test-older-k8s``:
+
+- For new branches: it triggers a ``Cilium-PR-Ginkgo-Tests-K8s`` which does exactly the same thing (triggers all ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs),
+  except that the compound job will registered in PR checks. This makes it more cumbersome to detect which specific version K8s failed in case of failure.
+- For older branches (where ``test-missed-k8s`` is unavailable): triggers a ``Cilium-PR-Ginkgo-Tests-K8s`` testing older K8s versions itself.
+  It should not be used for new branches, only for Backport PRs for which ``test-older-k8s`` is unavailable.
```

Comment:
Nit: Could you please wrap text on ~80 lines, to remain consistent with the rest of the document?

Reply:
Oh my bad, I did not see it was 80-col. Fixing that.